src/bundles/ApplicationsAdmin/components/EvidenceAssessment/EvidenceAssessment.js

```diff
@@ -0,0 +1,325 @@
+  }
+
+  hasReviewedAllCriteria() {
+    return Object.keys(this.state.criteria).every(id => {
+      return this.state.criteria[id].demonstrates === true || (this.state.criteria[id].demonstrates === false && this.state.criteria[id].reason)
+    })
+  }
+
+  hasReviewiedVFM() {
+    return this.state.vfm !== undefined
+  }
+
+  hasMetEnoughCriteria() {
+    return (
+      this.props.evidence.criteriaNeeded > 0 &&
+      Object.keys(this.state.criteria).filter(id => this.state.criteria[id].demonstrates === true).length >= this.props.evidence.criteriaNeeded
+    )
+  }
+
+  getCriteriaName(criteriaId) {
+    const match = this.props.evidence.domain_criteria.find(x => parseInt(x.id, 10) === parseInt(criteriaId, 10))
+    return match.name ? match.name : ''
+  }
+
+  getReasonOptions() {
+    const reasonsText = [
+      'Not clear what the seller specifically did',
+      'The evidence does not contain enough detail to assess',
+      'The evidence does not relate to the criteria',
+      'The evidence only partially demonstrates the criteria',
+      'The situation cited does not reflect a common government need'
+    ]
+    const reasons = [
+      {
+        value: '',
+        text: 'Select a reason...'
+      }
+    ]
+    reasonsText.map(reason => reasons.push({ value: reason, text: reason }))
+    return reasons
+  }
+
+  handleCriteriaReviewClick(e, criteriaId) {
+    e.persist()
+    this.setState(curState => {
+      const value = e.target.value === "yes" ? true : false
+      const newState = { ...curState }
+      newState.criteria[criteriaId].demonstrates = value
+      return newState
+    })
+  }
+
+  handleVFMClick(vfm) {
+    this.setState({
+      vfm
+    })
+  }
+
+  handleFeedbackChange(e, type, criteriaId) {
+    e.persist()
+    this.setState(curState => {
+      const feedback = e.target.value
+      const newState = { ...curState }
+      newState.criteria[criteriaId][type] = feedback
+      return newState
+    })
+  }
+
+  handleAssessmentReject() {
+    const evidenceId = this.props.match.params.id
+    const failed_criteria = {}
+    Object.keys(this.state.criteria).map(criteriaId => {
+      if (this.state.criteria[criteriaId].demonstrates === false && this.state.criteria[criteriaId].reason) {
+        failed_criteria[criteriaId] = {
+          reason: this.state.criteria[criteriaId].reason,
+          feedback: this.state.criteria[criteriaId].feedback
+        }
+      }
+    })
+    const vfm = this.state.vfm
+    this.props.rejectEvidence(evidenceId, failed_criteria, vfm).then(res => {
+      this.setState({
+        wasApproved: false,
+        wasRejected: true
+      })
+    })
+  }
+
+  handleAssessmentApprove() {
+    const evidenceId = this.props.match.params.id
+    this.props.approveEvidence(evidenceId).then(res => {
+      this.setState({
+        wasApproved: true,
+        wasRejected: false
+      })
+    })
+  }
+
+  priceIsOver() {
```

Comment:
small thing.. do we allow decimals in the price?

Reply:
I don't believe so (current profile edit pricing doesn't allow it).